lib/pure/os.nim

```diff
@@ -468,7 +388,7 @@ proc splitFile*(path: string): tuple[dir, name, ext: string] {.
   noSideEffect, rtl, extern: "nos$1".} =
   ## Splits a filename into `(dir, name, extension)` tuple.
   ##
-  ## `dir` does not end in `DirSep <#DirSep>`_.
+  ## `dir` does not end in `DirSep <#DirSep>`_ unless it is absolute.
```

Comment:
Misleading doc: So ... absolute paths can end in the slash? Why? Only true for the root directory "/" or what?

Reply:
fixed